<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PLC emulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTreeView" name="diView"/>
    </item>
    <item row="0" column="2">
     <widget class="QTreeView" name="dqView"/>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QTreeView" name="mView"/>
    </item>
    <item row="0" column="5">
     <widget class="QTreeView" name="tView"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTreeView" name="aiView"/>
    </item>
    <item row="1" column="2">
     <widget class="QTreeView" name="aqView"/>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QTreeView" name="rView"/>
    </item>
    <item row="1" column="5">
     <widget class="QTreeView" name="sView"/>
    </item>
    <item row="2" column="0" rowspan="3">
     <widget class="QListView" name="listView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>113</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2" rowspan="3" colspan="2">
     <widget class="QPlainTextEdit" name="programEdit"/>
    </item>
    <item row="2" column="4" rowspan="3" colspan="2">
     <widget class="QTextBrowser" name="consoleBrowser"/>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButtonAdd">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButtonRemove">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>PLC EMU v.2.1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>RUNNING</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QTimeEdit" name="timeEdit">
      <property name="currentSection">
       <enum>QDateTimeEdit::HourSection</enum>
      </property>
      <property name="displayFormat">
       <string>h:mm:ss AP</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionStart_stop"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionReadme"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionStart_stop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start </string>
   </property>
  </action>
  <action name="actionForce_Unforce">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Force </string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionReadme">
   <property name="text">
    <string>Readme</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connect </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>setBrowserSource()</slot>
 </slots>
</ui>
